components/centos/centos-openshift-setup/Vagrantfile

```diff
@@ -115,8 +115,8 @@ Vagrant.configure(2) do |config|
     echo "Configured cluster admin user: admin, Password: admin"
     echo
     echo "To use OpenShift CLI, run:"
-    echo "$ vagrant ssh"
-    echo "$ oc login"
+    echo "vagrant service-manager install-cli openshift"
```

Comment:
This may not be required if the user already has `oc` binary.

Probably, changing the wording with conditional phrase might make sense.